docs/swagger.yaml

```diff
@@ -334,12 +333,13 @@ components:
           description: Organisation number of the merchant for whom the partner is submitting the product order.
         salesUnitName:
           type: string
-          pattern: '^[a-zA-Z0-9- ]{1,30}$'
+          pattern: '^[\p{L}\p{M}0-9-_\s\.]{1,30}$'
```

Comment:
Is intended to not support a name like `Vipps+`?

Reply:
I don't really know, I'm open to suggestion, I tried to ensure support for the ones I could think of. Do a suggest change for if you can think of any more.